core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceResponseConverterTest.java

```diff
@@ -252,6 +254,7 @@ public Object expectOk() {
                 }
 
                 @Get("/expect-custom-header")
+                @AdditionalHeader(name = "x-custom-annotated-header", value = "annotated-value")
                 @ProducesJson
                 public HttpResult<Map<String, String>> expectCustomHeader() {
                     return HttpResult.of(HttpHeaders.of(AsciiString.of("x-custom-header"), "value"),
```

Comment:
Global comment: For header names, you should use `HttpHeaderNames.of(CharSequence)`.